dbms/src/test/java/nl/nn/adapterframework/dbms/H2DbmsSupportTest.java

```diff
@@ -1,12 +1,10 @@
-package nl.nn.adapterframework.jdbc.dbms;
+package nl.nn.adapterframework.dbms;
 
-import static org.junit.Assert.assertEquals;
+import org.junit.jupiter.api.Test;
 
 import java.sql.SQLException;
 
-import org.junit.Test;
-
-import nl.nn.adapterframework.jdbc.JdbcException;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Het ziet er naar uit dat volgorde van static en niet-static imports nu anders is?

Reply:
Dat is niet de bedoeling, ik pas het gelijk aan